src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,65 @@
+import classNames from 'classnames';
+import React, { useEffect } from 'react';
+
+import { ErrorType } from '../../enums/ErrorType';
+
+type Props = {
+  errorType: ErrorType;
+  isErrorShown: boolean;
+  onCloseNotification: () => void;
+};
+
+export const ErrorNotification: React.FC<Props> = React.memo(
+  ({ errorType, isErrorShown, onCloseNotification }) => {
+    let errorMessage = '';
+
+    switch (errorType) {
+      case ErrorType.Add:
+      case ErrorType.Delete:
+      case ErrorType.Update:
+      case ErrorType.Download:
+        errorMessage = `Unable to ${errorType} a todo`;
+
+        break;
+
+      case ErrorType.EmptyTitle:
+        errorMessage = 'Title can\'t be empty';
+
+        break;
+
+      default:
+        errorMessage = '';
+    }
```

Comment:
Use useState for the errorMessage value and wrap this switch/case to the useEffect, please

Reply:
But if I write like this and use setErrorMessage in useEffect, it will cause two renders instead of one, because the latter runs after the render and forces React to do another one